src/Command/AddMassMediaCommand.php

```diff
@@ -22,11 +25,26 @@
  */
 class AddMassMediaCommand extends BaseCommand
```

Comment:
We already introduced a BC break with the commands. So it's a good point where we remove the `BaseCommand` usage too

Reply:
Should I also add `final`??

Reply:
If we are using directly command we should leave the BaseCommand intact and just deprecate it. That will cause less BC breaks

Reply:
I agree, `BaseCommand` should be keep but should not be extended by other commands. I ask about `final` modifier. It can be add becouse this PR introduce BC-break. With it API can be close and internal code can be improve.

Reply:
IMO we should do the minimal BC break, and making this final is not needed

Reply:
Sorry, I was not talking about the removal of the entire class, I meant something like this:


```suggestion
class AddMassMediaCommand extends Command
```

Reply:
I know. IMHO we should keep it to reduce BC-break. We can remove it without any problem in next MAROJ where this classes will be final.